flink-java/src/test/java/org/apache/flink/api/java/sca/UdfAnalyzerTest.java

```diff
@@ -268,8 +274,8 @@ public void testForwardIntoNestedTupleWithVarAndModification() {
 
 	@Test
 	public void testForwardIntoTupleWithAssignment() {
-		compareAnalyzerResultWithAnnotationsSingleInput(MapFunction.class, Map14.class, "Tuple2<String,Integer>",
-				"Tuple2<String,String>");
+		compareAnalyzerResultWithAnnotationsSingleInput(MapFunction.class, Map14.class,
+			TypeInformation.of(new TypeHint<Tuple2<String, Integer>>(){}), TypeInformation.of(new TypeHint<Tuple2<String, String>>(){}));
```

Comment:
There is so much use of the types `Tuple2<String, Integer>>` and `Tuple2<String, String>`, it would make sense to factor these out into a static field and reference them from there.

Reply:
accept this suggestion